.idea/runConfigurations/DEBUG__Run_on_Device__Remote_JVM_.xml

```diff
@@ -4,7 +4,7 @@
     <option name="USE_SOCKET_TRANSPORT" value="true" />
     <option name="SERVER_MODE" value="false" />
     <option name="SHMEM_ADDRESS" />
-    <option name="HOST" value="172.20.20.6" />
+    <option name="HOST" value="10.0.0.151" />
```

Comment:
Looks like a runconfiguration update snuck in to this commit.

Reply:
If you use the IntelliJ Git integration, it is usually helpful to enable the "Group By > Directory" option so the files in .idea show up separately from everything else.